pxtlib/browserutils.ts

```diff
@@ -135,8 +135,20 @@ namespace pxt.BrowserUtils {
 
     let hasLoggedBrowser = false
 
+    export function isLocalStorageSupported(): boolean {
```

Comment:
This helper should be used in pxt.storage to test if localStorage can be used. Should be moved in new PR.